Control coverage/README.md

```diff
@@ -40,6 +40,7 @@ Below resource types can be checked for validating the security controls:
 |[VirtualMachine](Feature/VirtualMachine.md)|Microsoft.Compute/virtualMachines|
 |[VirtualMachineScaleSet](Feature/VirtualMachineScaleSet.md)|Microsoft.Compute/virtualMachineScaleSets|
 |[VirtualNetwork](Feature/VirtualNetwork.md)|Microsoft.Network/virtualNetworks|
+|[FrontDoor](Feature/FrontDoor.md)|Microsoft.Network/frontDoor|
```

Comment:
Please take latest from Main and adjust the item num accordingly.

Reply:
Please make sure that we have same num sequence

Reply:
took latest & changed there